sql/slow-query.md

```diff
@@ -68,19 +67,74 @@ select * from t_slim, t_wide where t_slim.c0=t_wide.c0;
 #### succ
 
 表示请求是否执行成功。
+=======
+* `Time`：表示日志打印时间。
+* `Txn_start_ts`：表示事务的开始时间戳，也是事务的 ID, 可以用这个值在日志中 grep 出事务相关的日志。
```

Comment:
- There are a lot of format issues in this section. Please pay attention to them and update them all.
- Actually it's unnecessary to merge these new changes in master to your branch. If you do, please guarantee the correctness of format and wording.

Reply:
@IzabelWang You have incorrectly merge outdated content into this PR. Please pay more attention to accuracy.